src/blockdata/constants.rs

```diff
@@ -112,7 +111,7 @@ pub fn genesis_block(network: Network) -> Block {
                 header: BlockHeader {
                     version: 1,
                     prev_blockhash: Default::default(),
-                    merkle_root: txdata.merkle_root(),
+                    merkle_root: txdata[0].txid(),
```

Comment:
If you implement MerkleRoot for &[Transaction] you shouldn't need to change this, no?

Reply:
I am not sure we should have MerkleRoot for &[Transaction] as it is with txid() for Block but with bitcoin_hash() for witness root.

Reply:
Ohoh, right, forgot it was used in two different contexts. Indeed.